content/_index.md

```diff
@@ -3,25 +3,40 @@ title: Kubernetes Contributors
 
-## Introduction
+### Get Involved:
 
-We can put anything we want here, news, Guin with cats, etc.
+The first step is to check out the [contributor guide](/guide/). All contributors must abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and [sign the CLA](https://github.com/kubernetes/community/blob/master/CLA.md).
 
-{{< youtube -GayuFobHBo >}}
+If you need help with submitting a pull request, found a bug, or have a suggestion, please join us in [Slack] and pop over to the [#kubernetes-dev channel][kdev].
 
-## Code Walkthrough
+Remember: No question is too small.
 
-If you are interested in contributing code, it is recommended that you begin
-with this code walkthrough by Stefan Schimanski
+### Stay Involved:
 
-{{< youtube yqB_le-N6EE >}}
+There are many ways you can stay involved with the Kubernetes Contributor community. We’d love to have you join a [SIG](https://git.k8s.io/community/sig-list.md), contribute to and explore our [developer guide](https://git.k8s.io/community/contributors/devel) and contributor guide, or hear your thoughts at one of our biweekly [contributor experience SIG meetings](https://github.com/kubernetes/community/tree/master/sig-contributor-experience#meetings). We welcome you to meet our contributors by joining our [monthly one hour meeting](https://github.com/kubernetes/community/blob/master/mentoring/meet-our-contributors.md), or by watching our Office Hours livestream on [YouTube](https://www.youtube.com/c/kubernetescommunity/live)!
+
+Learn more about our community by joining in our [weekly community meeting](https://github.com/kubernetes/community/blob/master/events/community-meeting.md)
+Take a journey to [becoming a reviewer or approver](http://git.k8s.io/community/community-membership.md)!
+Check out our entire developer guide in the [community repo](http://git.k8s.io/community/contributors/devel)
+Find out how we make decisions, organize, and how to get involved in [governance](http://git.k8s.io/community/governance.md)
+
+### Evolution of this site:
+
+We’re building out this site in real-time! Want to join us and help? Here’s what we have in store for next iterations:
+
+- An Events page showcasing all current and future happenings within the Kubernetes community. We hope to launch this feature by November 2019. Want to help us hit this target? Help us [work on this project](https://github.com/kubernetes-sigs/contributor-site/issues/15) by forking the repo and submitting a pull request!
+- Contributor guide/handbook: Feature launch date estimated November 2019
+- Role Board: Feature launch date estimated November 2019
+- Directory of Kubernetes SIGs and other community groups
+- Pathways to success for new Kubernetes contributors [link to community membership doc] and mentoring programs
```

Comment:
The link for the community membership doc is this:
https://github.com/kubernetes/community/blob/master/community-membership.md

Reply:
Done!